### PR TITLE
Updating vss-api-nuget package - catching Exception for blobstore client telemetry

### PR DESCRIPTION
**Pull request by fadnavistanmay**
Updating vss-api-nuget package - catching Exception for blobstore client telemetry

### CODE REVIEW

**Review by alex-peck — APPROVED** [review]
Wait until the package build is finished to merge, but this is good.

**Review by anatolybolshakov — APPROVED** [review]

**Pull request merged**
